### ISSUE TITLE
red in default light theme too subtle

### ISSUE THREAD

**Issue by jrieken** [Microsoft/vscode#4502]
Like quite a few other men (https://en.wikipedia.org/wiki/Color_blindness#Frequency_of_red-green_color_blindness_in_males_of_various_populations) I suffer from impaired red/green vision and for my eyes the red value `rgba(128, 0, 0, 1)` from our default light theme is almost impossible to tell apart from the default black.

**Comment by jrieken** [comment]
related to https://github.com/Microsoft/vscode/issues/4133

**Comment by bgashler1** [comment]
Good find, @jrieken .  This is not the only color that I've heard from users is in need of some tweaking.  I will enhance the contrast.

**Comment by isidorn** [comment]
Moving this to June milestone because it is not critical
@bgashler1 please be aware of issues which you assign to the current milestone as we plan to fix them in the current milestone. I moved this out to June because we are wrapping up May

**Comment by bgashler1** [comment]
## Problem
Trying to solve this color blindness issue for markdown header syntax (and possibly other places where this color value is used).

This is a simulation I created using color tools in Illustrator of Protanopia (color blindness where you can't see red)
![untitled picture](https://cloud.githubusercontent.com/assets/11839736/16508722/d7b852f2-3eea-11e6-8520-e2d24d47120c.png)

Notice the red headers look almost exactly the same as the black text.


This is only an issue in the light theme, not the dark theme, because we a nice blue in the dark theme which works with the two most common forms of color-blindness.

I like how Quiet Light theme solves this.

![untitled picture](https://cloud.githubusercontent.com/assets/11839736/16508728/ef9ba6c6-3eea-11e6-8009-b11d4ba799c3.png)

They bold their text, and it's a different shade of red.  It also helps you understand the hierarchy while you write it and the monospaced nature of the font is not compromised, so no alignment issues.

However, I don't want to change our color if possible, because it's used in hundreds of places in our code and is consistent with Visual Studio IDE.

So what if we just bolded our current color like Quiet Light?

![untitled picture](https://cloud.githubusercontent.com/assets/11839736/16508739/0123f33a-3eeb-11e6-8b44-77ba4f2b7eb0.png)


While it fixes it in the plain text, as you can see a header is not rendered bold in our preview of markdown.  So, bolding isn't actually literally how it will be styled, and this could be confusing potentially.

Also, we don't use bold in the dark theme, so we would need to apply it there as well.

Would changing the shade of the color, though, make a difference?

<img width="725" alt="screen shot 2016-06-30 at 5 49 55 pm" src="https://cloud.githubusercontent.com/assets/11839736/16508751/19290434-3eeb-11e6-9e15-461bd16fbddc.png">

Obviously, you can see that there is numerically more color present in all color channels (meaning more brightness to contrast against text).

But actually all colors are not showing up well in red color blindness.  Notice how even Quiet Light doesn't have enough contrast in their red when I take the boldness off of it on the headers.
<img width="462" alt="screen shot 2016-06-30 at 5 55 08 pm" src="https://cloud.githubusercontent.com/assets/11839736/16508834/d24a333e-3eeb-11e6-895f-2c3a48678a9b.png">


## Possible solutions
	1) Make bold (like Quiet Light theme)
	2) Make markdown headers same color as **bold** markdown token (which is like what we already do in the dark theme)
		a. This would leave other usage of this color unaddressed, but the only one reported so far that I know of is with markdown.  So it would buy us some time to reevaluate all our colors.
	3) Change all instances of this color
Difficult to do, and we should carefully test it in all situations.  We can't simply make the red more bright, because for example in HTML, this color is used to mark tags and another lighter shade of red is used to mark attributes (which sit right next to tags).

## My proposal
Go with option 2

**Comment by bgashler1** [comment]
This is what that option looks like...

**(simulating red-color-blindness)**
<img width="368" alt="screen shot 2016-06-30 at 6 06 04 pm" src="https://cloud.githubusercontent.com/assets/11839736/16508991/5a309c88-3eed-11e6-8e4d-e8bb9170f9c2.png">

**(regular)**
<img width="429" alt="screen shot 2016-06-30 at 6 08 45 pm" src="https://cloud.githubusercontent.com/assets/11839736/16509022/bc7f4574-3eed-11e6-985d-7c9f438e5115.png">

Pretty similar, huh?  I'm using `rgb(36, 36, 181)`, but we can of course discuss that.

**Comment by bgashler1** [comment]
I decided not to change the color of the heading but instead apply a bold font-weight (as is used in the Quiet Light theme).  If anyone disagrees, please let me know.  I think the bold color fixes this in a very conservative way by helping people who have trouble seeing the red to differentiate headings based on the boldness (it also helps everyone parse documents for headings faster with the heavier weight, regardless of their color perception).

Lastly, It also makes it easier for people to distinguish between *bold* and *headings* by having two different colors.  This is why I chose not to go with option 2 above.

Here is what it looks like with protanopia simulated...
<img width="356" alt="screen shot 2016-07-21 at 5 29 20 pm" src="https://cloud.githubusercontent.com/assets/11839736/17043283/0a7d43c0-4f6a-11e6-97f2-abf58dfabc74.png">
